metadata-ingestion/src/datahub/ingestion/source/sql/mssql/job_models.py

```diff
@@ -72,15 +72,15 @@ class MSSQLProceduresContainer:
 
     @property
     def formatted_name(self) -> str:
-        return f"{self.formatted_platform_instance}.{self.name.replace(',', '-')}"
+        return self.name.replace(',', '-')
 
     @property
     def orchestrator(self) -> str:
         return self.source
 
     @property
     def formatted_platform_instance(self) -> str:
-        return self.platform_instance.replace(".", "/")
+        return self.platform_instance
```

Comment:
looking at this in more detail, it seems like platform_instance really should be of type `Optional[str]` and not `str` as declared here